shared/js/background/storage/config.es6.js

```diff
@@ -0,0 +1,74 @@
+// Babel includes for async/await
+import 'regenerator-runtime/runtime'
+
+const load = require('./../load.es6')
+const constants = require('../../../data/constants')
+const settings = require('./../settings.es6')
+
+/**
+ *  Manage local cookie exclusion data.
+ *  Data is updated regularly with the latest
+ *  cookie exclusions, and stored here for fast retrieval and usage
+ *  when checking which cookies to block.
+ **/
+class ConfigStorage {
```

Comment:
What's the reasoning behind creating a new class and loading logic for this file when tdsStorage is already able to load and persist files to disk (including protections.json which this replaces)?